src/api/auth/AuthProvider.tsx

```diff
@@ -64,12 +64,20 @@ const AuthProvider: FC<{ children: ReactNode }> = ({ children }) => {
     setCookies(response.data);
   }
 
+  async function logout() {
+    console.log('Logout called')
+    clearCookies();
+    setAuthenticated(false);
+    window.location.href = buildLogoutHref();
+  }
+
```

Comment:
Non-blocking, but important: would you be amenable to us adding a backlog issue to research revoking tokens as well to ensure misconfigured or browsers on compromised systems do not "get logged out" when they think they are? This peeves me in personal contexts, and it seems AWS might support it.

https://docs.aws.amazon.com/cognito/latest/developerguide/token-revocation.html

I'd like to spike and consider this, after reviving our threat model. Thoughts? /cc @Compton-NIST

Reply:
To be clear, I am worried about the refresh tokens since those are the focus of the revocation, but their official docs are confused by AWS devs saying how Cognito operates against API calls, so maybe I am overreacting. 🤷